mlflow/data.py

```diff
@@ -75,3 +76,14 @@ def download_uri(uri, output_path):
     else:
         raise DownloadException("`uri` must be a DBFS (%s), S3 (%s), or GCS (%s) URI, got "
                                 "%s" % (DBFS_PREFIX, S3_PREFIX, GS_PREFIX, uri))
+
+
+def get_local_path(path_or_uri):
```

Comment:
From offline discussion: should we move this to `file_utils` instead? Seems a little confusing to have these path-to-or-from-uri-conversion utils scattered around modules